ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/lacp-configuration.j2

```diff
@@ -0,0 +1,25 @@
+{% if lacp is arista.avd.defined %}
+
+# LACP
+
+## LACP Summary
+
+| Port-id range | Rate-limit | System-priority |
```

Comment:
If you change the heading to `Rate-limit default` you can just use the boolean value directly with default to `-`

Reply:
Rate-limit is default. We can only set it to no default.